feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
+
```

Comment:
it is really good that you have decided this way

Reply:
thanks for this comment